packages/cashier/src/_common/socket_base.js

```diff
@@ -0,0 +1,400 @@
+const DerivAPIBasic = require('@deriv/deriv-api/dist/DerivAPIBasic');
```

Comment:
why do we need `socket_base.js` and `socket_cache.js` in cashier package?
Is it because of the dependency in `server_time.js`? If so, can we import `WS` from `ws-method.js` in `server_time.js`?

Reply:
Thanks for the tip. Looks like we can replace it with WS methods. I've added the changes. Please check again.